test/unit/math/fwd/mat/fun/qr_thin_R_test.cpp

```diff
@@ -0,0 +1,72 @@
+#include <stan/math/fwd/mat.hpp>
+#include <gtest/gtest.h>
+
+TEST(AgradFwdMatrixQrR, fd) {
+  using stan::math::matrix_d;
+  using stan::math::matrix_fd;
+  matrix_fd m0(0, 0);
+  matrix_d m2(3, 2);
+  matrix_fd m1(3, 2);
+  m1 << 1, 2, 3, 4, 5, 6;
+  m2 << 1, 2, 3, 4, 5, 6;
+  m1(0, 0).d_ = 1.0;
```

Comment:
Any idea what the meaning of this test is? All the .d_s set to 1.0? My issue with it is I wouldn't know how to check res(0, 0).d_ offhand without a bunch of thinking, but I guess since this came from the existing qr_R stuff it's fine. It's certainly not the partial of the output with respect to one parameter.